orca-deploymentmonitor/src/main/java/com/netflix/spinnaker/config/DeploymentMonitorServiceProvider.java

```diff
@@ -75,8 +75,7 @@ private synchronized DeploymentMonitorService getServiceByDefinition(
               .setRequestInterceptor(spinnakerRequestInterceptor)
               .setEndpoint(definition.getBaseUrl())
               .setClient(retrofitClient)
-              // TODO(mvulfson): Change logging level
-              .setLogLevel(RestAdapter.LogLevel.HEADERS_AND_ARGS)
+              .setLogLevel(RestAdapter.LogLevel.BASIC)
```

Comment:
I think there is a `retrofitLogLevel` `@Bean` floating around the spring context that makes this value configurable via yaml

Reply:
yeah, i was trying to find it but didn't look hard enough, let me look more harder... 
one thing i discovered... is that retrofit logging (outside of basic) kinda sorta sucks... hence i did it myway so i can actually find it/read it/use it